assets/javascripts/lib/discourse-markdown/bbcode-plugin.js

```diff
@@ -75,6 +75,7 @@ export function setup(helper) {
     "div.bb-border",
     "div.bb-center",
     "div.bb-check",
+    "div.bb-height-restrict",
     "div.bb-justify",
```

Comment:
`div.bb-img` isn't in the allowList here, which means it would get cleansed from the final HTML once everything is cooked. Are you sure the image tag styling needs to be/is being added? Just want to make sure what you expect is going in.